src/main.rs

```diff
@@ -33,12 +33,14 @@ fn main() -> anyhow::Result<()> {
             .long("offline")
             .global(true)
             .help("No network call made by elm-test-rs"),
+        Arg::with_name("verbose")
+            .short("v")
+            .multiple(true)
+            .global(true)
+            .help("Increase verbosity. Can be used multiple times -vv"),
```

Comment:
```suggestion
            .help("Increase verbosity. Can be used multiple times -vvv"),
```

`-vvv` is the maximum, right? I think it makes sense to show that. Users can figure out that `-vv` is ok too?

Reply:
Right! makes sense